index.html

```diff
@@ -76,6 +76,10 @@
 			<div style="clear:both"></div>
 		</nav>
 
+		<section class="banner">
+            <h1>UGA Hacks 7 October 22-24</h1>
```

Comment:
Love the look of the banner - For the wording, maybe we can expand on it like 'UGAHacks 7 is happening on October 22-24. Interested in joining us? (linked to the interest form)' or something similar yet concise enough to be easy to view on mobile. Could also make the tag an h2 in that case